src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -113,12 +114,27 @@ private string BuildModuleKey(string projectGuid)
             await SynchronizeSuppressedIssues();
         }
 
+        private Task DoInitialFetch()
```

Comment:
This is one potential fix. As discussed, I'll try @Evangelink's PR that disables the Connect command to see if that would be an alternative.